src/app/forms/templates/hgv/hgv-tech-record.template.ts

```diff
@@ -9,121 +15,150 @@ export const HgvTechRecord: FormNode = {
       name: 'vehicleType',
       label: 'Vehicle type',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.VEHICLETYPE
+      viewType: FormNodeViewTypes.VEHICLETYPE,
+      editType: FormNodeEditTypes.SELECT,
+      options: [
+        { value: 'psv', label: 'PSV' },
+        { value: 'hgv', label: 'HGV' },
+        { value: 'trl', label: 'Trailer' }
+      ],
     },
     {
       name: 'regnDate',
       label: 'Date of first registration',
       value: '',
-
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.DATE
+      viewType: FormNodeViewTypes.DATE,
+      editType: FormNodeEditTypes.DATE,
+      validators: [{ name: ValidatorNames.Required }],
+      isoDate: false
     },
     {
       name: 'manufactureYear',
       label: 'Year of manufacture',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.TEXT,
+      validators: [
+        { name: ValidatorNames.MaxLength, args: 4 },
+        { name: ValidatorNames.Required }
+      ]
     },
     {
       name: 'noOfAxles',
       label: 'Number of axles',
       value: '',
-
+      width: FormNodeWidth.XXS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      validators: [{ name: ValidatorNames.Required }],
     },
     {
       name: 'brakes',
       label: 'DTP number',
       value: '',
+      type: FormNodeTypes.GROUP,
       children: [
         {
           name: 'dtpNumber',
           label: 'DTP number',
           value: '',
-
+          width: FormNodeWidth.M,
           type: FormNodeTypes.CONTROL,
-          viewType: FormNodeViewTypes.STRING
+          validators: [{ name: ValidatorNames.Required }],
         }
       ],
-      type: FormNodeTypes.GROUP,
-      viewType: FormNodeViewTypes.STRING
-    },
-    {
-      name: 'axles',
-      value: '',
-      type: FormNodeTypes.ARRAY,
-      children: [
-        {
-          name: '0',
-          label: 'Axle',
-          value: '',
-          type: FormNodeTypes.GROUP,
-          children: [
-            {
-              name: 'parkingBrakeMrk',
-              label: 'Axles fitted with a parking brake',
-              value: '',
-
-              type: FormNodeTypes.CONTROL,
-              viewType: FormNodeViewTypes.STRING
-            }
-          ]
-        }
-      ]
     },
+    // {   /// Not on UI model but may be needed later
```

Comment:
I'd remove all the sections of comment-out code. Especially if we don't know if it will ever be needed.

Reply:
It will probably be needed as soon as the wider axle management problem is resolved